pdf.js

```diff
@@ -1,21 +1,21 @@
-const jsreport = require('jsreport-core')()
+const pdf = require('html-pdf');
 
 module.exports = (req, res) => {
-  jsreport.init().then(function () {     
-    
-    jsreport.render({
-      template: {
-        content: req.body.content,
-        engine: 'jsrender',
-        recipe: 'phantom-pdf'
-      }
-    }).then((resp) => {
-      res.status(200).json({
-        data: resp.content.toString()
-      })
-    }).catch(function(e) {
-      console.log(e)
-    })
+    var html = req.body.content;
+    var options = {format: 'Letter', orientation: 'portrait', border: {
+        "top": "2cm",
+        "right": "1.5cm",
+        "bottom": "2cm",
+        "left": "1.5cm"
+    }};
+
+    pdf.create(html, options).toFile('PDF/prova.pdf', function(err, response){
```

Comment:
para o nome do arquivo, gerar um nome unico, pra quando gerar mais de um, talvez, não deixar exposto ID, fazer algum tratamento a partir do Id talvez, uma combinação de id e data, mas não de forma direta

Reply:
aí ele vai gerar arquivos diferentes e não sobreescrever o msm, se a gente for subir em algum lugar e tirar dai blz mas do contrario não é valido, vai começar a gerar arquivos no diretório da api.
Uma solução talvez seria ao inves de gravar em um arquivo fazer stream dele